source/upgrade.txt

```diff
@@ -102,6 +102,12 @@ Version 6.x Breaking Changes
   perform any operations in the session.
 - The ``includeResultMetadata`` option for compound operation methods is
   ``false`` by default.
+- The ``withSession()`` method returns the value that the provided
+  function returns. In previous driver versions, this function returns
+  ``void``. The ``withTransaction()`` method returns the value that the
```

Comment:
Suggestion:
I could be wrong, but I don't remember "void" being a valid Javascript type. I think it returns "undefined" and that it uses the void operator in the method signature:
- [withSession() API doc from v5.0](https://mongodb.github.io/node-mongodb-native/5.0/classes/MongoClient.html#withSession)
- ["A void function returns undefined"](https://stackoverflow.com/questions/44216949/does-a-async-function-which-returns-promisevoid-have-an-implicit-return-at-the)

Reply:
https://github.com/mongodb/node-mongodb-native/releases/#:~:text=Previously%2C%20this%20function%20returned%20void%20this%20is%20a%20feature%20to%20align%20with%20the%20following%20breaking%20change. Uses the wording "returns void" -- so I'm not sure.